target/linux/kirkwood/base-files/etc/board.d/01_leds

```diff
@@ -12,6 +12,9 @@ case "$board" in
 iom,ix2-200)
 	ucidef_set_led_timer "health" "health" "status:white:rebuild_led" "200" "800"
 	;;
+lenovo-ix2-dl)
+	ucidef_set_led_timer "power" "power" "status:white:rebuild_led" "200" "800"
+        ;;
```

Comment:
broken indent.

Using ucidef_set_led_timer is discouraged for new devices. Please use led-* aliases instead like all newer devices.